webapp/src/components/EventStatus.js

```diff
@@ -387,6 +405,11 @@ class EventStatus extends Component {
                 <span>{definition.title}</span><br/>{this.renderButton(definition)}
             </div>
         }
+        else if (definition.shortText && definition.link && this.props.submissionLink && definition.title !== definition.shortText) {
```

Comment:
This code seems unreachable to me? The previous condition has to be true in order for this to be true, plus the thing being returned is the same?

Reply:
Okay so you don't need this condition because I am already checking for submissionLink in renderButton.